easybuild/easyconfigs/l/LUSCUS/LUSCUS-0.8.6-foss-2018b.eb

```diff
@@ -29,7 +33,9 @@ dependencies = [
 
 sanity_check_paths = {
     'files': ['bin/luscus'],
-    'dirs': [],
+    'dirs': ['config'],
 }
 
+modextravars = {'LUSCUS_DIR': '%(installdir)s/config'}
```

Comment:
why not `modextrapath`?

Reply:
Because it's a pointer to a single directory, it's not a `$PATH`-like variable